Makefile

```diff
@@ -64,8 +64,8 @@ ifneq ($(shell echo $(APP_VERSION) | egrep '^v([0-9]+\.){0,2}(\*|[0-9]+)'),)
 endif
 
 ## Docker Images
-DOCKER_IMAGE_GO         ?= "golang:${GO_VERSION}@sha256:d83472f1ab5712a6b2b816dc811e46155e844ddc02f5f5952e72c6deedafed77"
-DOCKER_IMAGE_GOLINT     ?= "golangci/golangci-lint:v1.57.2@sha256:8f3a60a00a83bb7d599d2e028ac0c3573dc2b9ec0842590f1c2e59781c821da7"
+DOCKER_IMAGE_GO         ?= "golang:${GO_VERSION}@sha256:21edeab9ed48e9820f0b447cce6ce1900b3aa90ffce3c8b4de7fae5ac333de0c"
+DOCKER_IMAGE_GOLINT     ?= "golangci/golangci-lint:v1.62.2@sha256:0f3af3929517ed4afa1f1bcba4eae827296017720e08ecd5c68b9f0640bc310d"
```

Comment:
Is it necessary to update golangci-lint as part of this PR? If not, I'd like to keep it separate.

Reply:
As I understood `mlog.Logger` requires `go1.22`, for this reason I updated the images

Reply:
Hmm .. I see. So golangci 1.57 does not work with Go 1.21?

Reply:
maybe do you mean Go 1.22? Yes, it works, so I downgraded to 1.57.2

Reply:
The issue is with test, tests don't not compile with docker 1.21